### PR TITLE
Hotkeys

### PR DESCRIPTION
**Pull request by MajinBui**
Added functions and hotkeys for play/pause and stop/reset.  Some placeholders are used until further instructions.

### CODE REVIEW

File: keymaps/pomodoro-app.cson [hunk 1]
```diff
@@ -9,3 +9,6 @@
 # https://atom.io/docs/latest/behind-atom-keymaps-in-depth
 # 'atom-workspace':
 #   'ctrl-alt-o': 'pomodoro-app:toggle'
+'atom-workspace':
+  'ctrl-alt-j': 'atom-pomodoro-app:toggle'
```
Comment (apigarage) [inline keymaps/pomodoro-app.cson]: for consistency, how about naming it "toggleTimer"?

**Comment by MajinBui** [comment]
Currently, I have cleaned up the code to using a enum timerStateEnum:
https://github.com/MajinBui/atom-pomodoro-app/blob/hotkeys/lib/index.coffee#L26-L29

I also added buttons to the view:
https://github.com/MajinBui/atom-pomodoro-app/blob/hotkeys/lib/pomodoro-app-view.coffee#L10-L27
button-pomodoro

I am now trying to add event listeners to the button but it seems as though I might have to move the timer functions into the view.

**Comment by MajinBui** [comment]
Never mind, I was able to add the event listeners in for the buttons.  Now, all that is left is the actual timer itself.  Preview:
working-buttons
https://github.com/MajinBui/atom-pomodoro-app/blob/hotkeys/lib/index.coffee#L38-L39

**Comment by MajinBui** [comment]
TImer is complete.
timer completed

**Comment by MajinBui** [comment]
Yep, code related to debugging is now removed.

**Pull request merged**